File: JavaScript/프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.js
function solution(l, r) {
    let answer = [];
    for (let n=l; n<=r; n++){
        const number = String(n)
        let flag = true;
        for (const i of number){
            flag = true;
            if (i == '5' || i == '0'){
                continue;
            }else{
                flag = false;
                break;
            }
        }
        if (flag === true) {
            answer.push(n);
        }
    }
    if (answer == false) {
        answer.push(-1);
    }
    return answer;
}

File: C++/프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int l, int r) {
    vector<int> answer;
    bool flag = true;
    
    for (int i=l; i<=r; i++){
        string n = to_string(i);
        flag = true;
        for (int j=0; j<n.length(); j++){
            if (n[j] != '5' && n[j] != '0') {
                flag = false;
                break;
            }
        }
        
        if (flag) {answer.push_back(i);}
    }
    if (answer.size() == 0) {answer = {-1};}
    
    return answer;
}